plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/security/realms/ldap/internal/connector/dao/DynamicGroupsIT.java

```diff
@@ -198,19 +194,19 @@ public void testGroupManagerWithDynamicGroups()
     }
   }
 
+  @Test
   public void testGroupManagerWithDynamicGroupsDisabled()
       throws Exception
   {
-
-    Map<String, Object> env = new HashMap<String, Object>();
+    Map<String, Object> env = Maps.newHashMap();
     // Create a new context pointing to the overseas partition
     env.put(Context.INITIAL_CONTEXT_FACTORY, "com.sun.jndi.ldap.LdapCtxFactory");
-    env.put(Context.PROVIDER_URL, "ldap://localhost:" + this.getLdapServer().getPort() + "/o=sonatype");
+    env.put(Context.PROVIDER_URL, "ldap://localhost:" + ldapServers.get("default").getPort() + "/o=sonatype");
```

Comment:
Not essential, but would be beneficial to extract the duplicated logic for creation of InitialLdapContext. Makes it clear that we're using the same general setup for each test